events/README.md

```diff
@@ -0,0 +1,93 @@
+# Events
+
+This directory contains branding for events celebrated by Python Discord. Events are automatically discovered by the [Python](https://github.com/python-discord/bot/) bot and have their assets applied to the Discord guild when active. As such, event directories must follow a specific structure.
+
+## Event directory structure
+
+In order for an event directory to be valid, it has to contain the following assets.
```

Comment:
```suggestion
For an event directory to be valid, it has to contain the following assets.
```
`in order for` seems a bit wordy here, it is grammatically correct, but keeping it concise is better.